### DISCUSSION TITLE
Wrong attributed charge type (DC instead of AC) when car charging without signal

### DISCUSSION THREAD

**Discussion by enricotoso96** [teslamate-org/teslamate#1282] [Ideas]
I often charge at work where they recently installed some 11 kW AC chargers. Unfortunately the garage is underground and there is no cellular signal (and Teslas cannot connect to WPA2 Enterprise wifis yet). Teslamate reports these charges as DC fast charges instead of AC charges. Has anyone else seen this phenomenon? 

Could we get the option to change the type (DC or AC) of charge in the web app? Or is there a way to change this through the file system? Could be cool to be able to select the charger type in the geolocation settings..

**Comment by 1920x1200** [comment]
Did you ever figure this out? I'm having the same issue.
